src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -252,9 +339,33 @@ describe('claimant response converter ', () => {
         })
     })
 
-    it(' Reject court decision and refer to judge ', () => {
+    it(' Reject court decision to pay by set date and refer to judge ', () => {
+      const draftClaimantResponse = createDraftClaimantResponseWithCourtDecisionType(intentionOfImmediatePayment,
+        DecisionType.COURT, FormaliseRepaymentPlanOption.REFER_TO_JUDGE, paymentIntentionInInstallment,paymentIntentionBySetDate)
+
+      draftClaimantResponse.rejectionReason = new RejectionReason('rejected reason')
+      expect(converter.convertToClaimantResponse(draftClaimantResponse)).to.deep.eq(
+        {
+          'type': 'ACCEPTATION',
```

Comment:
can we move all these JSON examples to a data file? please look in test/data/entity for examples of files containing different JSON data objects that be reused across tests.